build.sbt

```diff
@@ -236,6 +236,7 @@ runSteward := Def.taskDyn {
   val args = Seq(
     Seq("--workspace", s"$projectDir/workspace"),
     Seq("--repos-file", s"$projectDir/repos.md"),
+    Seq("--default-repo-config", s"$projectDir/default.scala-steward.conf"),
```

Comment:
should this be `--default-repo-conf` sans `ig` ?

Reply:
Yes, good catch. I'll update it.